app/admin/views.py

```diff
@@ -200,24 +201,27 @@ def destinations_show(uuid):
 
     if edit_form.validate_on_submit():
         if edit_form.submit.data:
-            dest.name = edit_form.name.data,
-            dest.address = edit_form.address.data,
+            dest.name = edit_form.name.data
+            dest.address = edit_form.address.data
             dest.point = 'SRID=4326;POINT({} {})'.format(
-                edit_form.destination_lon.data,
-                edit_form.destination_lat.data
-            ),
+                edit_form.destination_lon.data, edit_form.destination_lat.data)
+
+            for carpool in dest.carpools:
+                carpool.to_place = dest.address
+                carpool.to_point = dest.point
+
             db.session.commit()
             flash("Your destination was updated", 'success')
-            return redirect(url_for('admin.destinations_show', uuid=uuid))
+            return redirect(url_for('admin.destinations_list'))
         elif edit_form.delete.data:
```

Comment:
I know this change wasn't made as part of this PR, but could you add a link to the delete page rather than a form button? It shouldn't be handled as part of form processing.

Reply:
@iandees  So from the list of destinations, for each destination, have a link to edit and a link to delete?

Reply:
No, have a delete link on the detail/show/edit page for a destination. Same as it is now, but use an `<a href>` instead of a form button.

Reply:
Oh.  Ok.